config/local.json.example

```diff
@@ -26,13 +26,16 @@
       "app": "chronicle",
       "debug": false,
       "fmt": "pretty",
-      "level": "verbose"
+      "level": "info"
```

Comment:
the server is ridiculously chatty on 'verbose', so it probably should not be the default level.